### ISSUE TITLE
Can't export data after patch

### ISSUE THREAD

**Issue by AnasElInmortal** [Trivadis/plsql-formatter-settings#275]
After patch dbtools-common.jar with SQLcl 23.4.0 cant export data query in sqlDeveloper 23.1.1.345

**Comment by PhilippSalvisberg** [comment]
Yes, the export produces empty files with the patched `db-common.jar`. Thanks for letting us know.

Today the SQLDev team presented this roadmap for [Oracle SQL Developer Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=Oracle.sql-developer):

![image](https://github.com/Trivadis/plsql-formatter-settings/assets/3883171/f6ae670b-c545-4a9f-9c7c-07b6e3fe3720)

Custom Format Settings are not mentioned. 

SQL Developer "Classic" will be end of life in 2026. However, SQL Developer "Classic" is de facto EOL, even if we can expect a bug-fix update here and there.

In other words, we have to stop using the custom format settings in SQL Developer "Classic" and wait until the Oracle SQL Developer Extension for VS Code supports custom format settings in a compatible way.

**Comment by WayneNani** [comment]
It seems like formatting [might be just around the corner](https://marketplace.visualstudio.com/items?itemName=Oracle.sql-developer&ssr=false#qna). But the "Advanced" section might take some time and especially input.
